android/sample/src/debug/java/com/facebook/flipper/sample/FlipperInitializer.java

```diff
@@ -57,7 +57,7 @@ public static IntializationResult initFlipperPlugins(Context context, FlipperCli
 
     final OkHttpClient okHttpClient =
         new OkHttpClient.Builder()
-            .addNetworkInterceptor(interceptor)
+            .addInterceptor(interceptor)
```

Comment:
Is it possible to use network interceptor? There's many cases which app have custom interceptor, we need to mock the ultimate request.

Reply:
Can't we just add it as the first interceptor. In that case, it will be the first interceptor.